WikiDBUpdaterHttp/WikiDBUpdaterHttp.cs

```diff
@@ -0,0 +1,144 @@
+                return new BadRequestObjectResult("Missing parameter: date");
+            if (!validateDate(YYYYMMDD, out startDate))
+                return new BadRequestObjectResult("Bad starting date: " + YYYYMMDD);
+
+            int daysToGo = 1;
+            string daysToGoString = req.Query["days"];
+            if (daysToGoString != null)
+                if (!int.TryParse(daysToGoString, out daysToGo))
+                    return new BadRequestObjectResult("Bad days parameter: " + daysToGoString);
+
+
+            string[] languages = allLanguageCodes;
+            string languageStrings = req.Query["languages"];
+            if (languageStrings != null)
+                if (!validateLanguages(languageStrings, out languages))
+                    return new BadRequestObjectResult("Bad languages parameter: " + languageStrings);
+
+
+            string discardString = req.Query["discard"];
+            bool discardOldData = discardString != null ? true : false;
+
+            IDBClient dbClient = new AzureStorageClient(tableClient);
+
+            await DBUpdates.updateDatabase(startDate, daysToGo, discardOldData, languages, 
+                articleExceptions, httpClient, dbClient, log);
+            
+            return new OkObjectResult("Successfull execution");
+        }
+
+        /// <summary>
+        /// Parses the YYYY-MM-DD string into DateTime
+        /// </summary>
+        /// <param name="_yyyymmdd">Date in a YYYY-MM-DD format</param>
+        /// <param name="_date">out for a date</param>
+        /// <returns>true if date was correctly parsed</returns>
+        bool validateDate(string _yyyymmdd, out DateTime _date)
+        {
+            _date = DateTime.Now;
+            int year, month, day;
+            string[] dateArray = _yyyymmdd.Split('-');
+
+            if (dateArray.Length != 3)
+                return false;
+
+            if (!int.TryParse(dateArray[0], out year))
+                return false;
+            if (!int.TryParse(dateArray[1], out month))
+                return false;
+            if (!int.TryParse(dateArray[2], out day))
+                return false;
+
+            _date = new DateTime(year, month, day);
+            return true;
+        }
+
+       
+        /// <summary>
+        /// Validates list of languages in "","","" format
+        /// </summary>
+        /// <param name="_language">string in "","","", format</param>
+        /// <param name="_languageArray">out for an output array</param>
+        /// <returns>returns true if string parsed into array</returns>
+        bool validateLanguages(string _language, out string[] _languageArray)
+        {
+            _languageArray = _language.ToLower().Split(',');
+            foreach (var iLanguage in _languageArray)
+            {
+                if (!allLanguageCodes.Contains(iLanguage))
+                {
+                    return false;
+                }
+            }
+            return true;
```

Comment:
```suggestion
            return _languageArray.All(language => allLanguageCodes.Contains(language));
```
LINQ is love, LINQ is life! 
Seriously, LINQ is a killer feature and you should use it.